templates/repo/issue/view_content.tmpl

```diff
@@ -19,6 +19,7 @@
 					<div class="ui top attached header">
 						<span class="text grey"><a {{if gt .Issue.Poster.ID 0}}href="{{.Issue.Poster.HomeLink}}"{{end}}>{{.Issue.Poster.Name}}</a> {{.i18n.Tr "repo.issues.commented_at" .Issue.HashTag $createdStr | Safe}}</span>
 						<div class="ui right actions">
+							{{template "repo/issue/view_content/reactions" Dict "ctx" $ "ActionURL" (Printf "%s/issues/%d/reactions" $.RepoLink .Issue.Index) "Reactions" .GetReactionsByType }}
```

Comment:
`.GetReactionsByType` is not defined in the current context, so `.GetReactionsByType` will evaluate to nil.

In this case, it gets us what we want (we *don't* want to show the current reactions in the issue header, just the button for adding reactions, and the reactions template won't render the current reactions if `Reactions` is nil), but this is confusing and non-obvious.

Can we make this more straightforward? My proposal would be to have two separate templates: one for displaying the current reaction, and another template for the button/pop-out to add new reactions. In the issue header, we would only use the second template, but in other places we would use both.